.github/workflows/publish.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish NPM Package
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish-npm-package-to-npm:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    outputs:
+      package-version: ${{ steps.package-version.outputs.current-version }}
+    name: Publish NPM Package
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+      - name: Set Current Package Version as a Variable
+        id: package-version   # Can be referenced with ${{ steps.package-version.outputs.current-version }}
+        uses: martinbeentjes/npm-get-version-action@main
```

Comment:
@beanc16 Do we still need to fetch the version in the publish action? It should automatically get it from the `package.json`, right?